### DISCUSSION TITLE
Team routing algorithm (load balancing) is not assigning to a particular team member.

### DISCUSSION THREAD

**Discussion by Moohan** [community/community#53132] [Pull Requests]
### Select Topic Area

Bug

### Body

We have a team set up with 5 members including myself, one of the members is specifically excluded from the assignment using the options on the team page. Otherwise, we have it set to load balance and assign it to two members. ![image](https://user-images.githubusercontent.com/5982260/232829405-afce6bac-39ec-4e25-92ba-947103b6c2dc.png) However, whenever I assign a PR review to the team it always picks the same 2 (out of 3 possible) people. This seemed a coincidence at first but it is still assigning it to the same 2 people without fail, even though they have multiple PRs to review and the third person has none.

I'm at a loss for what to try here but it's very annoying!

**Comment by simon-hogg** [comment]
@Moohan did you get any answer or help with this? I am stuck with I believe the same issue. "load-balancing" seeming to favour a certain 2 or 3 people in a team of 10 inexplicably.

**Reply by Moohan** [reply]
Nope no answer. I'm on a different team now and we tend to use CODEOWNNERS so I'm not sure if it still affects the other repo...

**Comment by mert-batu** [comment]


**Reply by simon-hogg** [reply]
Thank you for this detail explanation 🙌 

> Their GitHub notifications are disabled for the repo

I wonder if this might interfere with certain users setup. For example, I have specifically disabled notifications (except for  discussions) for a repo in order that I may receive Reminders via my Slack integration. This is so I only get notified for Review requests, mentions, and comments for that repo.

**Comment by anastasialanz** [comment]
I'm seeing the same issue. A co-worker kept being assigned PRs even when they had a bunch to review already.
